e2e/context/testdata/context-ls.golden

```diff
@@ -1,3 +1,3 @@
-NAME        DESCRIPTION                               DOCKER ENDPOINT               KUBERNETES ENDPOINT            ORCHESTRATOR
-default *   Current DOCKER_HOST based configuration   unix:///var/run/docker.sock   https://someserver (zoinx)     swarm
-remote      my remote cluster                         ssh://someserver              https://someserver (default)   kubernetes
+NAME        TYPE      DESCRIPTION                               DOCKER ENDPOINT               KUBERNETES ENDPOINT            ORCHESTRATOR
```

Comment:
Can you update the example output in https://github.com/docker/cli/blob/master/docs/reference/commandline/context_ls.md#examples to match the new format?

Reply:
Probably the `docker context inspect` example output should also be updated; https://github.com/docker/cli/blob/master/docs/reference/commandline/context_inspect.md#inspect-a-context-by-name

Reply:
Done